basic_samples/DataViews/Python3/README.md

```diff
@@ -29,8 +29,6 @@ or
 
 Automated test uses Python 3.6.8 x64
 
-Complete dependencies listed in [DEPENDENCIES.md](DEPENDENCIES.md)
```

Comment:
we should probably include a LICENSE file that has the complete dependency list: using the blackduck readout https://osisoft.blackducksoftware.com/api/projects/2f22e568-1ba9-453f-864b-332bad4c4aa3/versions/5254b5b7-bf30-4937-9fbc-4d50a9880d53/reports

Reply:
Moved LICENSE file to the right folder (had the wrong casing)